openstack/clientconfig/requests.go

```diff
@@ -18,6 +19,14 @@ type ClientOpts struct {
 
 	// EnvPrefix allows a custom environment variable prefix to be used.
 	EnvPrefix string
+
+	// AuthType specifies the type of authentication to use.
+	// By default, this is "password".
+	AuthType string
```

Comment:
Can we make this a more specific type than `string`?

Reply:
Yes, good catch.